src/sqlfluff/dialects/dialect_ansi.py

```diff
@@ -2378,7 +2378,7 @@ class CTEDefinitionSegment(BaseSegment):
     match_grammar: Matchable = Sequence(
         Ref("SingleIdentifierGrammar"),
         Ref("CTEColumnList", optional=True),
-        "AS",
+        Sequence("AS", optional=True),
```

Comment:
Would this work? It seems strange to use `Sequence` for a single segment unless there's a reason to.

```suggestion
        Ref("AS", optional=True),
```